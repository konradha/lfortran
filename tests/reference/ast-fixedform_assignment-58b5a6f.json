{
    "basename": "ast-fixedform_assignment-58b5a6f",
    "cmd": "lfortran --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixedform_assignment.f",
    "infile_hash": "a5ce54b54cc0b1865c8561b13812ea8490888bb2807c4a7bddd5ba39",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-fixedform_assignment-58b5a6f.stdout",
    "stdout_hash": "22d5aaf358249a80a6ebaeae6fcc0461f25e9b19699de0020a17ed83",
    "stderr": "ast-fixedform_assignment-58b5a6f.stderr",
    "stderr_hash": "3426c44bf98464a0a067cb6360023a06c4495008a54a6337d97f0660",
    "returncode": 0
}