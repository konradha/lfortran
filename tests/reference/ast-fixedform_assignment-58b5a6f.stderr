style suggestion: Use 'end if' instead of 'endif'
  --> tests/fixedform_assignment.f:15:7
   |
15 |       endif
   |       ^^^^^^ help: write this as 'end if'
